### ISSUE TITLE
[AAPT2] Encountering new error  CS0117: 'Resource' does not contain a definition for 'Foo' when enabling aapt2

### ISSUE THREAD

**Issue by pjcollins** [xamarin/xamarin-android#1983]
I'm encountering a new class of error that occur when enabling aapt2 that doesn't appear to be a direct result of an aapt2 invocation.

### Steps to Reproduce

1. Build the following project(s) with aapt2 enabled:
* https://github.com/xamarin/monodroid-samples/tree/master/ApplicationFundamentals/FingerPaint
* https://github.com/xamarin/monodroid-samples/tree/master/SearchableDictionary

### Expected Behavior
Projects compile with Aapt2 enabled.

### Actual Behavior
```
MainActivity.cs(29,74): error CS0117: 'Resource' does not contain a definition for 'Array' [/Users/xamarinqa/agent/_work/r1/a/monodroid-samples/ApplicationFundamentals/FingerPaint/FingerPaint/FingerPaint.csproj]
MainActivity.cs(37,80): error CS0117: 'Resource' does not contain a definition for 'Array' [/Users/xamarinqa/agent/_work/r1/a/monodroid-samples/ApplicationFundamentals/FingerPaint/FingerPaint/FingerPaint.csproj]
```
```
DictionaryDatabase.cs(138,59): error CS0117: 'Resource' does not contain a definition for 'Raw' [/Users/xamarinqa/agent/_work/r1/a/monodroid-samples/SearchableDictionary/SearchableDictionary.csproj]
```

### Version Information
XA version: 9.0.0.17 (HEAD/dfb09269d) 

### Log File
http://xqa.blob.core.windows.net/gist/log-7b4e8e9639004445816d5920c549229b.txt
http://xqa.blob.core.windows.net/gist/log-fb8b38fbbb894b4ba26c074dc86df622.txt

**Comment by dellis1972** [comment]
Related PR https://github.com/xamarin/xamarin-android/pull/2014

**Comment by dellis1972** [comment]
Fixed Via #2014

**Comment by dellis1972** [comment]
Related PR https://github.com/xamarin/xamarin-android/pull/2045
